semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/types/expressions/generalizations/AstNodeTraversal.scala

```diff
@@ -49,6 +49,16 @@ class AstNodeTraversal[A <: AstNode](val traversal: Traversal[A]) extends AnyVal
   def astParent: Traversal[AstNode] =
     traversal.in(EdgeTypes.AST).cast[AstNode]
 
+  /** Traverses up the AST and returns the first block node.
+    */
+  def parentBlock: Traversal[Block] =
+    traversal
+      .repeat(_.in(EdgeTypes.AST))(
+        _.emit
+          .until(_.hasLabel(NodeTypes.BLOCK))
+      )
+      .collectAll[Block]
```

Comment:
```suggestion
    traversal.repeat(_.in(EdgeTypes.AST))(
      _.emit.until(_.hasLabel(NodeTypes.BLOCK))
    ).collectAll[Block]
```

matter of personal preference - feel free to disregard